osu.Game/Graphics/Backgrounds/BeatmapBackgroundWithStoryboard.cs

```diff
@@ -33,18 +42,52 @@ public BeatmapBackgroundWithStoryboard(WorkingBeatmap beatmap, string fallbackTe
         [BackgroundDependencyLoader]
         private void load()
         {
+            AddInternal(storyboardContainer = new AudioContainer
+            {
+                RelativeSizeAxes = Axes.Both,
+                Volume = { Value = 0 },
+            });
+
+            LoadStoryboard();
```

Comment:
`LoadStoryboard()` has a `LoadComponentAsync()` call inside, so this doesn't look all that correct? BDL is already in an async context / on a separate load thread, so this will just take up another thread to load the storyboard but let the BDL method terminate.

Reply:
This is how it used to exist beforehand, but I'll fix.

Reply:
Applied in https://github.com/ppy/osu/pull/25400/commits/bb912bc6161a8785f105e3da29f7e95f83261249